pyvista/plotting/renderer.py

```diff
@@ -3676,17 +3676,21 @@ def add_legend(
             * ``'upper center'``
             * ``'center'``
 
-        face : str | pyvista.PolyData | NoneType, default: "triangle"
-            Face shape of legend face.  One of the following:
+        face : str | pyvista.PolyData, optional
```

Comment:
```suggestion
        face : str | pyvista.PolyData, optional, default: "triangle"
```
This expression is still need.

Reply:
The default isn't triangle though. If glyphs are input (as in our glyph legend test):

```py
def test_legend_using_add_legend_only_labels(random_hills, verify_image_cache):
    pl = pv.Plotter()

    arrows = random_hills.glyph(scale="Normals", orient="Normals", tolerance=0.05)
    pl.add_mesh(arrows, color="black", label="label 1")

    pl.add_mesh(random_hills, scalars="Elevation", cmap="terrain", show_scalar_bar=False)

    legend_entries = ["label 1", "label 2"]

    pl.add_legend(legend_entries, size=(0.5, 0.5))
    pl.show()
```
![image](https://github.com/pyvista/pyvista/assets/11981631/69094c0f-4fcd-4f3d-a417-925421d161ac)


The default isn't triangle (nor is the default text).

Reply:
Thanks for explanation. Got it :)